app/views/products/_form.html.erb

```diff
@@ -8,7 +8,11 @@
   <%= number_field_tag(:quantity, @product.quantity, min: 0, step: 1) %>
   <%= label_tag(:categories, "Categories") %>
   <%= select_tag "categories", options_for_select(@categories, params["categories"]), include_blank: "none", :multiple => true %>
-  <%= label_tag(:category, "Category") %>
```

Comment:
We should leave these 2 lines in until we have another way to add a custom category